Aztec/Classes/TextKit/MediaAttachment.swift

```diff
@@ -227,18 +216,18 @@ open class MediaAttachment: NSTextAttachment {
         return glyphImage
     }
 
-    func mediaBounds(forBounds bounds: CGRect) -> CGRect {
+    func mediaBounds(for bounds: CGRect) -> CGRect {
         let containerWidth = bounds.size.width
         let origin = CGPoint(x: xPosition(forContainerWidth: bounds.size.width), y: imageMargin)
         let size = CGSize(width: onScreenWidth(containerWidth), height: onScreenHeight(containerWidth) - imageMargin)
         return CGRect(origin: origin, size: size)
     }
 
-    fileprivate func glyph(basedOnImage image:UIImage, forBounds bounds: CGRect) -> UIImage? {
+    private func glyph(basedOnImage image:UIImage, forBounds bounds: CGRect) -> UIImage? {
```

Comment:
To stay consistent `basedOnImage` -> `for`, and probably `forBounds` -> `in` or `constrainedTo`.